request-deadline-decorator.js

```diff
@@ -0,0 +1,18 @@
+exports.version = 1;
+
+exports.druidRequestDecoratorFactory = function (logger, params) {
+  return function (decoratorRequest, context) {
```

Comment:
FYI: You can actually use most / all es6 features here. Typescript does not pre compile the file but node supports it natively